src/target/mod.rs

```diff
@@ -665,6 +665,15 @@ pub trait Target {
     fn support_auxv(&mut self) -> Option<ext::auxv::AuxvOps<'_, Self>> {
         None
     }
+
+    /// Support for querying register info in LLDB.
```

Comment:
```suggestion
    /// (LLDB specific) Alternate method of querying register info.
```

I haven't decided on how `gdbstub`'s long-term LLDB compat story is going to look like, but for now, I'd lean towards marking any "entrypoints" into LLDB-specific sections of the codebase with `(LLDB specific)`. This applies to the mod-comment in `register_info.rs` as well.

Reply:
Ahh, sorry. It seems that the wording that's currently being used is actually `(LLDB extension)`, so please use that instead.

See https://github.com/daniel5151/gdbstub/pull/100#discussion_r811520963 for the last time this came up.